docs/troubleshooting.rst

```diff
@@ -106,17 +106,18 @@ You may experience these errors when using the optional TIKA integration:
 Gotenberg is a server that converts Office documents into PDF documents and has a default timeout of 30 seconds.
 When conversion takes longer, Gotenberg raises this error.
 
-You can increase the timeout by configuring an environment variable for Gotenberg (see also `here <https://gotenberg.dev/docs/modules/api#properties>`__).
+You can increase the timeout by configuring a command flag for Gotenberg (see also `here <https://gotenberg.dev/docs/modules/api#properties>`__).
 If using docker-compose, this is achieved by the following configuration change in the ``docker-compose.yml`` file:
 
 .. code:: yaml
 
     gotenberg:
         image: gotenberg/gotenberg:7
         restart: unless-stopped
-        environment:
-            CHROMIUM_DISABLE_ROUTES: 1
-            API_PROCESS_TIMEOUT: 60
+        command:
+            - "gotenberg"
+            - "--chromium-disable-routes=true"
+            - "--api-process-timeout=60"
```

Comment:
[The docs state](https://gotenberg.dev/docs/modules/api) ("Timeout" tab):

> The properties --api-read-timeout, --api-process-timeout and --api-write-timeout are deprecated in favor of --api-timeout.

Should we use the latter?

Reply:
Thanks for your review! Will make the requested changed on the upcomming weekend as I am not near my gear right now.

Reply:
@tribut I made the requested change. Sorry, it took so long. Unfortunately, I cannot resolve the review comment after anymore, because I updated the existing commit. Can you help out and have a look?